platform-enterprise_docs/getting-started/production-checklist.md

```diff
@@ -0,0 +1,94 @@
+
+## Organizations and workspaces
+
+Organizations are the top-level structure and contain workspaces, members, and teams.
+
+You can create multiple organizations within Seqera Platform, each of which can contain multiple workspaces with shared users and resources. This means you can customize and organize the use of resources while maintaining an access control layer for users associated with a workspace.
+
+Before you create your organizations and workspaces, consider the various roles and work streams that you’d like to start with and scale to. See [Organizations](https://docs.seqera.io/platform-cloud/orgs-and-teams/organizations) for more information.
+
+## Users and roles
+
+Once an organization is created, the user who created the organization is the default owner of that organization. Additional users can be assigned as organization owners. Owners have full read/write access to modify members, teams, collaborators, and settings within an organization.
+Organization members have specific roles and permissions that define their access and capabilities:
+
+- A member is a user who is internal to the organization.
+- Members have an organization role and can operate in one or more organization workspaces.
+- In each workspace, members have a participant role that defines the permissions granted to them within that workspace.
+- Members can create their own workspaces within an organization and be part of a team.
+
+All users can be assigned roles that grant the type of access and permissions they have to resources within Platform. 
+
+It’s a good idea to map out the expected users and their roles to ensure your plans are scalable. See [User roles](https://docs.seqera.io/platform-enterprise/25.1/orgs-and-teams/roles) for more information.
+
+## Infrastructure
+
+Infrastructural requirements vary widely depending on the workload you expect. See [Enterprise installation](https://docs.seqera.io/platform-enterprise/latest/enterprise/overview) for an outline of Platform components.
+
+To begin, build out a proof of concept using the below recommendations and create a baseline of your capacity requirements. When you are ready to move to production, consider the increased workload.
+
+### Kubernetes
+
+When deploying Seqera Platform in a generic Kubernetes cluster we recommend starting with: 
+
+- 4 vCPU
+- 7 GB nodes 
+
+This sizing recommendation is a basic starting point. Your requirements may vary significantly based on the number of pipelines and concurrent processes you anticipate. See [Configure Pods and Containers](https://kubernetes.io/docs/tasks/configure-pod-container/) for information about increasing your resources.
+
+### Docker
+
+When deploying Seqera Platform using Docker compose we recommend starting with: 
+
+- Instance size - `c5.2xlarge`
+- External DB Aurora V3 provisioned - `db.t3.medium`
+- External Redis - `cache.t2.micro`
```

Comment:
Minor comment, non blocking, each section here "Docker", "AWS" ect, has a different style

E.g.,

Instance size - `c5.2xlarge`
Instance size: `c5.2xlarge`
Instance size set to `c5.2xlarge`